js/count_characters.js

```diff
@@ -38,4 +70,71 @@ function addCount(obj) {
 function getParagraphs() {
     var paragraphs = document.getElementsByClassName("exchange--speaker");
     return paragraphs;
-};
+}
+
+
+
+function update_subtitle(){
+    var current_time = time_bar.getElementsByClassName("timecode text--noselect")[0];
+    var paragraphs =  getParagraphs();
+    var index = get_index_of_paragraph(paragraphs, current_time.innerText);
+    parag_text = as_text(paragraphs[index]);
+    if (!is_complete(parag_text)){
+        next_parag_text = as_text(paragraphs[index+1]);
```

Comment:
Please handle `out of index` error here when the last paragraph has `###` at the end of it.

Reply:
Done